readme.md

```diff
@@ -9,16 +9,16 @@ the C source of an older version of the compiler are needed to bootstrap the
 latest version. The C sources are available in a separate repo [here](http://github.com/nim-lang/csources).
 
 Pre-compiled snapshots of the compiler are also available on
-[Nimbuild](http://build.nim-lang.org/). Your platform however may not 
+[Nimbuild](http://buildbot.nim-lang.org/). Your platform however may not
```

Comment:
Also, I don't think it provides pre-compiled snapshots. So maybe you should just remove it?